lib/pymedphys/.pylintrc

```diff
@@ -43,6 +43,9 @@ disable =
     possibly-used-before-assignment,
     useless-import-alias,
 
+[DESIGN]
+max-positional-arguments=20
```

Comment:
Pylint have introduced a new check for too many positional args - see changelog [here](https://pylint.pycqa.org/en/stable/whatsnew/3/3.3/index.html#new-checks).

Just added this line to fix for now, but open to alternative approaches.